### DISCUSSION TITLE
Border color in menu hover-bordered

### DISCUSSION THREAD

**Discussion by GeertHauwaerts** [saadeghi/daisyui#1471] [Q&A] [answered]
How do I change the `hover-bordered` color from `primary` to, for example, `error`?

For example, following https://daisyui.com/components/menu/#item-with-border-on-hover, I want the purple to become red.

**Comment by GeertHauwaerts** [comment]
To clarify, I only want 1 specific menu-item to have a different color (it's for a delete action)

**Comment by saadeghi** [comment, marked as answer]
One way would be adding `hover:!border-error` class to the item inside `<li>`  
(you can use any other color instead of `error`)
Example: https://play.tailwindcss.com/1IwQ4q17ig

Soon I will apply some changes that the `!` wouldn't be required for this modification.

Other than that, you can add a custom CSS  
Or you can change the color variable for that specific `<li>` to your custom color using a class name like `[--p:0_100%_50%]` (these numbers are HSL values): https://play.tailwindcss.com/onzWmtmsPU

**Comment by GeertHauwaerts** [comment]
Thanks! Works perfectly :)
